src/Network/Network/help/Set-AzNetworkManager.md

```diff
@@ -0,0 +1,225 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version:
+schema: 2.0.0
+---
+
+# Set-AzNetworkManager
+
+## SYNOPSIS
+Updates a network manager..
+
+## SYNTAX
+
+```
+Set-AzNetworkManager -ResourceGroupName <String> -InputObject <PSNetworkManager> [-AsJob]
```

Comment:
I think the ResourceGroupName can be obtained from InputObject, can it?

Reply:
Done for all resources